core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -37,6 +42,9 @@ public function get_loader_scripts() {
 			// Apps.
 			'elementor-editor',
 			'elementor-packages-editor',
+
+			// Utils.
+			'elementor-packages-ui',
```

Comment:
why is it here? it's enqueued automatically like the locations

Reply:
Removed.